compiler/rustc_resolve/src/late.rs

```diff
@@ -4669,14 +4741,15 @@ impl<'a, 'ast, 'ra, 'tcx> LateResolutionVisitor<'a, 'ast, 'ra, 'tcx> {
             self.ribs[ValueNS].push(Rib::new(RibKind::Normal));
         }
 
+        self.ribs[ValueNS]
+            .push(Rib::new(RibKind::Block { id: block.id, is_module: is_module_block }));
```

Comment:
```suggestion
            .push(Rib::new(RibKind::Block { id: block.id, is_module }));
```
Nit: rename for brevity.